tests/test_timeseriesflattener/test_add_values.py

```diff
@@ -233,3 +235,79 @@ def test_add_age_error():
             id_to_date_of_birth_mapping=str_to_df(static_predictor),
             date_of_birth_col_name="date_of_birth",
         )
+
+
+def test_low_variance_threshold_outcome():
+    prediction_times_df_str = """dw_ek_borger,timestamp,
+                            1,2021-12-31 00:00:00
+                            1,2022-01-02 00:00:00
+                            5,2025-01-02 00:00:00
+                            5,2025-08-05 00:00:00
+                            """
+    outcome_df_str = """dw_ek_borger,timestamp,value,
+                        1,2021-12-31 00:00:01, 1.0
+                        1,2023-01-02 00:00:00, 1.0
+                        5,2025-01-03 00:00:00, 1.0
+                        5,2022-01-05 00:00:01, 1.0
+                        """
+
+    with pytest.raises(KeyError):
```

Comment:
Oh, since pytest 3.1 there's a new "match" argument:

```with pytest.raises(ValueError, match=r'REGEX HERE'):```

We can use that to test if it matches the correct error :-)